index.html

```diff
@@ -232,4 +232,36 @@ <h3>Skills</h3>
             </div>
         </section>
     </section>
+    <section class="footer">
```

Comment:
- [ ] Instead of `<section>` element here, Please use `<footer>` tag. It will make sure your code is semantic and follows HTML best practices.

Reply:
resolved